mpd/mpd.go

```diff
@@ -660,8 +660,7 @@ func (m *MPD) AddNewAdaptationSetSubtitleWithID(id string, mimeType string, lang
 // label - Label for the subtitle from Studio (i.e. American)
 func (period *Period) AddNewAdaptationSetSubtitle(mimeType string, lang string, label string) (*AdaptationSet, error) {
 	as := &AdaptationSet{
-		Lang:  Strptr(lang),
-		Label: Strptr(label),
```

Comment:
Rather than removing this entirely, we should change to use the new field
`Labels: []string{label},` and the test fixtures should have the new label element added

Reply:
Agreed. Would be nice to see all the fixtures that currently have the attribute set to instead be updated to use the element.